bitrise.yml

```diff
@@ -1,166 +1,42 @@
+  sample:
+    envs:
+    - TEST_APP_URL: https://github.com/bitrise-samples/sample-apps-ios-simple-objc-with-uitest.git
+    - TEST_APP_BRANCH: renamed-scheme
+    - BITRISE_PROJECT_PATH: ios-simple-objc/ios-simple-objc.xcodeproj
+    - BITRISE_SCHEME: renamed_scheme
     steps:
     - script:
         inputs:
         - content: |-
-            #!/bin/bash
+            #!/bin/env bash
             set -ex
-            rm -rf "$ORIG_BITRISE_SOURCE_DIR/_tmp"
-            mkdir -p "$ORIG_BITRISE_SOURCE_DIR/_tmp"
-    - change-workdir:
-        title: Switch working dir to _tmp dir
-        run_if: true
-        inputs:
-        - path: "$ORIG_BITRISE_SOURCE_DIR/_tmp"
-        - is_create_path: true
+            rm -rf "./_tmp"
     - git::https://github.com/bitrise-steplib/bitrise-step-simple-git-clone.git:
         inputs:
-        - repository_url: $SAMPLE_APP_URL
-        - clone_into_dir: .
-        - branch: $SAMPLE_APP_BRANCH
-        - commit: $SAMPLE_APP_COMMIT
-    - certificate-and-profile-installer:
-
-  _output_test:
-    steps:
-    - script:
-        title: Output check
-        inputs:
-        - content: |-
-            #!/bin/bash
-            set -e
-            if [[ "$BITRISE_DEPLOY_DIR/testbundle.zip" != "$BITRISE_TEST_BUNDLE_ZIP_PATH" ]] ; then 
-              echo "BITRISE_TEST_BUNDLE_ZIP_PATH (\"$BITRISE_TEST_BUNDLE_ZIP_PATH\") should be: \"$BITRISE_DEPLOY_DIR/testbundle.zip\""
-              exit 1
-            fi
-            if [[ "$BITRISE_DEPLOY_DIR/Debug-iphoneos" != "$BITRISE_TEST_DIR_PATH" ]] ; then 
-              echo "BITRISE_TEST_DIR_PATH (\"$BITRISE_TEST_DIR_PATH\") should be: \"$BITRISE_DEPLOY_DIR/Debug-iphoneos\""
-              exit 1
-            fi
-            if [[ "$BITRISE_XCTESTRUN_FILE_PATH" =~ "$BITRISE_DEPLOY_DIR/${SCHEME}_iphoneos[0-9]{2}.[0-9]{1,2}-arm64e.xctestrun" ]] ; then 
-              echo "BITRISE_XCTESTRUN_FILE_PATH (\"$BITRISE_XCTESTRUN_FILE_PATH\") should be: \"$BITRISE_DEPLOY_DIR/${SCHEME}_iphoneos{{sdk-version}}-arm64e.xctestrun\""
-              exit 1
-            fi
-
-            echo "BITRISE_TEST_BUNDLE_ZIP_PATH: $BITRISE_TEST_BUNDLE_ZIP_PATH"
-            echo "BITRISE_TEST_DIR_PATH: $BITRISE_TEST_DIR_PATH"
-            echo "BITRISE_XCTESTRUN_FILE_PATH: $BITRISE_XCTESTRUN_FILE_PATH"
-
-  go-tests:
-    steps:
-    - go-list:
-    - golint:
-    - errcheck:
-    - go-test:
-
-  dep-update:
-    title: Dep update
-    description: |
-      Used for updating bitrise dependencies with dep
-    steps:
-    - script:
-        title: Dependency update
+        - repository_url: $TEST_APP_URL
+        - branch: $TEST_APP_BRANCH
+        - clone_into_dir: ./_tmp
+    - path::./:
+        title: Step Test
         inputs:
-        - content: |-
-            #!/bin/bash
-            set -ex
-            go get -u -v github.com/golang/dep/cmd/dep
-            dep ensure -v
-            dep ensure -v -update
+        # Inputs with non-default value or using environment variables
```

Comment:
Is this comment needed?

Reply:
we have this in [xcode-test step's bitrise.yml](https://github.com/bitrise-steplib/steps-xcode-test/blob/master/bitrise.yml#L39), I thought it can be handy (for contributors) to reassure that the step has many other inputs and we only list the ones with non-default value.